### PR TITLE
Configure XML secure processing

### PR DESCRIPTION
**Pull request by matthias-ronge**
Fixes a vulnerability reported by SonarCloud. For details, see [here](https://sonarcloud.io/organizations/henning-gerhardt-github/rules?open=squid%3AS2755&rule_key=squid%3AS2755).

### CODE REVIEW

**Comment by Beacze** [comment]
Our processing is not secure. You can't use it for working with kitodo.xml or meta.xml.

**Comment by matthias-ronge** [comment]
The link explains that this change simply disallows loading XML entities from other files. We don’t use that in any of our XMLs, so there shouldn’t be problems for us.

**Comment by Beacze** [comment]
Our files itself are treated as unsafe.

File: OpacPica-Plugin/src/main/java/org/kitodo/production/plugin/opac/pica/GetOpac.java [hunk 1]
```diff
@@ -16,16 +16,17 @@
...
 
+import javax.xml.XMLConstants;
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
 import javax.xml.parsers.SAXParserFactory;
 
 import org.apache.commons.httpclient.HttpClient;
 import org.apache.commons.httpclient.methods.GetMethod;
-import org.apache.logging.log4j.LogManager;
```
Comment (henning-gerhardt) [inline OpacPica-Plugin/src/main/java/org/kitodo/production/plugin/opac/pica/GetOpac.java]: Why did you switch from log4j version 2 logging instance to an other? This change breaks the compilation of your pull request.

Comment (matthias-ronge) [reply OpacPica-Plugin/src/main/java/org/kitodo/production/plugin/opac/pica/GetOpac.java]: Oh, that's why! The import organizes Eclipse on its own, and somehow it has a habit of changing that occasionally. Why, I have not found out yet.

**Comment by matthias-ronge** [comment]
Yes, that's exactly what I want to change. Or is there something against it? We do not use any XML entities.

File: OpacPica-Plugin/src/main/java/org/kitodo/production/plugin/opac/pica/GetOpac.java (hunk 1 above)
Comment (Kathrin-Huber) [reply OpacPica-Plugin/src/main/java/org/kitodo/production/plugin/opac/pica/GetOpac.java]: but reorganizing the imports shouldn't change the source, right?
If really eclipse is doing this, you should absolutely find the setting and disable it, beacuse this is not acceptable! Furthermore you should check your Pull Requests to identify such behaviour.

**Pull request merged**